defaults/test_default.yaml

```diff
@@ -54,7 +54,6 @@ post_behavior_loader_nodes: "destroy"
 post_behavior_monitor_nodes: "keep-on-failure"
 post_behavior_k8s_cluster: "keep-on-failure"
 
-cloud_credentials_path: '~/.ssh/support'
```

Comment:
If this is removed, let's remove it completely from sct_config

Reply:
If not, let's leave an empty default.

Reply:
OK, I'll set it to empty string.